cfy_manager/components/postgresql_server/config/supervisord/rsyslog.conf

```diff
@@ -1,4 +1,4 @@
 [program:rsyslog]
 startsecs=0
 umask=0066
-command=/usr/sbin/rsyslogd -n
\ No newline at end of file
+command=/etc/syslog_wrapper_script.sh
```

Comment:
Scripts don't go in /etc. If it's one that we're providing it can probably go somewhere under the /opt/cloudify tree

Reply:
sure will update that